action.yml

```diff
@@ -522,7 +554,7 @@ runs:
         echo "CA_GH_ACTION_USE_UP=${CA_GH_ACTION_USE_UP}" >> $GITHUB_ENV
 
     - name: Update the Container Registry details on the existing Container App
-      if: ${{ env.CA_GH_ACTION_ACR_LOGIN_ARG != '' && env.CA_GH_ACTION_USE_UP != 'true' && env.CA_GH_ACTION_RESOURCE_EXISTS == 'true' && env.CA_GH_ACTION_YAML_PROVIDED != 'true' }}
+      if: ${{ env.CA_GH_ACTION_REGISTRY_LOGIN_ARG != '' && env.CA_GH_ACTION_USE_UP != 'true' && env.CA_GH_ACTION_RESOURCE_EXISTS == 'true' && env.CA_GH_ACTION_YAML_PROVIDED != 'true' }}
       shell: bash
       run: |
         az containerapp registry set \
```

Comment:
We're still using `acrName`, `acrUsername` and `acrPassword` below in the call to this `az containerapp registry set` command -- we need to introduce new registry environment variables that can be used for this step. We can set these environment variables in the same step that `CA_GH_ACTION_REGISTRY_LOGIN_ARG` is created for each scenario as it's guaranteed that all three values required will be provided as input values to the action.